backend/server.py

```diff
@@ -590,15 +587,21 @@ def get_user(user_id):
     # Convert the user document to a dictionary
     user_data = user_doc.to_dict()
 
-    # Convert the date to string
+    # Convert the bio and username to strings
     user_data["bio"] = str(user_data["bio"])

```

Comment:
I don't think that these need to be strings. They should already be the right types when we pass them in.

Reply:
I will refactor and resolve this redundant conversion